build/Common.props

```diff
@@ -20,6 +20,7 @@
     <TargetFrameworksForAspNetCoreInstrumentation>net7.0;net6.0;netstandard2.1;netstandard2.0</TargetFrameworksForAspNetCoreInstrumentation>
     <TargetFrameworksForGrpcNetClientInstrumentation>net6.0;netstandard2.1;netstandard2.0</TargetFrameworksForGrpcNetClientInstrumentation>
     <TargetFrameworksForPrometheusAspNetCore>net6.0</TargetFrameworksForPrometheusAspNetCore>
+    <TargetFrameworksForTests>net8.0;net7.0;net6.0</TargetFrameworksForTests>
```

Comment:
Might fit better in Common.nonprod.props?

Reply:
@xiang17 FYI https://github.com/open-telemetry/opentelemetry-dotnet/pull/4904#discussion_r1341771973

Reply:
This is now defined at Ln33 (introduced in #4907).